lib/msal-common/src/utils/Metadata.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import metadataJson from "./metadata.json";
```

Comment:
Importing from .json files has been problematic in the past. Can we save the data to a .ts file instead?

Reply:
We are generating this `json` file directly from the response, this would be difficult to make into a .ts file
What problems have we faced in the past?

Reply:
I have added a typescript file for the raw metadata, I believe I encountered the problem you mentioned on building the project with a JSON file.